apps/meteor/tests/e2e/channel-management.spec.ts

```diff
@@ -76,6 +76,63 @@ test.describe.serial('channel-management', () => {
 		await expect(poHomeChannel.toastSuccess).toBeVisible();
 	});
 
+	test('should ignore "user1" messages', async ({ browser }) => {
+		await poHomeChannel.sidenav.openChat(targetChannel);
+		await poHomeChannel.tabs.btnTabMembers.click();
+		await poHomeChannel.tabs.members.showAllUsers();
+		await poHomeChannel.tabs.members.ignoreUser('user1');
+
+		await expect(poHomeChannel.toastSuccess).toBeVisible();
+
+		const user1Page = await browser.newPage({ storageState: Users.user1.state });
+		const user1Channel = new HomeChannel(user1Page);
+		await user1Page.goto(`/channel/${targetChannel}`);
+		await user1Channel.waitForChannel();
+		await user1Channel.content.sendMessage('hello');
+
+		await expect(poHomeChannel.content.lastUserMessageBody).toContainText('This message was ignored');
+
+		await user1Page.close();
+	});
+
+	test('should uniggnore single "user1" message', async ({ browser }) => {
+		await poHomeChannel.sidenav.openChat(targetChannel);
+
+		const user1Page = await browser.newPage({ storageState: Users.user1.state });
+		const user1Channel = new HomeChannel(user1Page);
+		await user1Page.goto(`/channel/${targetChannel}`);
+		await user1Channel.waitForChannel();
+		await user1Channel.content.sendMessage('only message to be unignored');
+
+		await poHomeChannel.sidenav.openChat(targetChannel);
+
+		await expect(poHomeChannel.content.lastUserMessageBody).toContainText('This message was ignored');
+		await poHomeChannel.content.lastIgnoredUserMessage.click();
+		await expect(poHomeChannel.content.lastUserMessageBody).toContainText('only message to be unignored');
+
+		await user1Page.close();
+	});
+
+
+	test('should unignore "user1" messages', async ({ browser }) => {
+		await poHomeChannel.sidenav.openChat(targetChannel);
+		await poHomeChannel.tabs.btnTabMembers.click();
+		await poHomeChannel.tabs.members.showAllUsers();
+		await poHomeChannel.tabs.members.unignoreUser('user1');
```

Comment:
Maybe we should send a message before this just to ensure the user is still ignored. 🤔

Reply:
Thank you! I added a new validation to ensure the user is still ignored and I added one that checks to see if the previous message is also being unignored